src/public/CowSwap.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "CoW Swap",
-  "timestamp": "2023-02-22T16:00:00+00:00",
+  "timestamp": "2023-03-22T12:00:00+00:00",
   "version": {
     "major": 2,
-    "minor": 0,
+    "minor": 1,
     "patch": 0
```

Comment:
Minor version bump as we are only adding tokens. See https://github.com/Uniswap/token-lists#semantic-versioning